charts/external-dns/templates/deployment.yaml

```diff
@@ -126,7 +130,49 @@ spec:
           resources:
             {{- toYaml . | nindent 12 }}
           {{- end }}
-      {{- if or .Values.secretConfiguration.enabled .Values.extraVolumes }}
+        {{- if .Values.webhook.image }}
+        - name: webhook
+          {{- with .Values.securityContext }}
+          securityContext:
+            {{- toYaml . | nindent 12 }}
+          {{- end }}
+          image: {{ .Values.webhook.image . }}
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          {{- with .Values.webhook.env }}
+          env:
+            {{- toYaml . | nindent 12 }}
+          {{- end }}
+          args:
+          {{- range .Values.webhook.args }}
+            - {{ tpl . $ }}
+          {{- end }}
+          ports:
+            - name: webhook-metrics
+              protocol: TCP
+              containerPort: 8080
+          livenessProbe:
+            {{- toYaml .Values.webhook.livenessProbe | nindent 12 }}
+          readinessProbe:
+            {{- toYaml .Values.webhook.readinessProbe | nindent 12 }}
+          {{- if or .Values.webhook.secretConfiguration.enabled .Values.webhook.extraVolumeMounts }}
```

Comment:
`.Values.webhook.secretConfiguration.enabled` is not in the `values.yaml`

Reply:
Changed to use `mountPath` instead of `enabled`, to match the external-dns container and the condition two lines below.

This does mean that setting either `mountPath` without also setting `enabled` will result in mounting a nonexistent volume. We could make the condition for the volume definition be an `or` of the two `mountPath`, leaving `enabled` to just control creating the `Secret`.